### PR TITLE
Decimal charge handling

### PR DESCRIPTION
**Pull request by aaronk**
- Adds support for decimal_charge handling by creating decimal_charge specific versions of IFXBillingRecordList, IFXBillingRecordTransactions and IFXBillingRecordHeader.  The proper IFXBillingRecordList variant is selected on a per-facility basis- currently just 'Research Computing Storage'
- Adds decimal_charge to BillingRecord and BillingTransaction
- Adds "dollar" filter that does not translation from pennies

### CODE REVIEW

File: src/components/billingRecord/IFXBillingRecordList.vue [hunk 1]
```diff
@@ -544,7 +549,7 @@ export default {
 
         this.editingIndex = index
         this.editedRecord = cloneDeep(item)
-        this.newExpenseCode = this.$api.account.create(item.account)
+        this.newExpenseCode = await this.$api.account.create(item.account)
```
Comment (luthian) [inline src/components/billingRecord/IFXBillingRecordList.vue:549]: Is `await` needed here?

Comment (aaronk) [reply src/components/billingRecord/IFXBillingRecordList.vue:549]: Yes.

File: src/components/billingRecord/IFXBillingRecordList.vue [hunk 2]
```diff
@@ -495,9 +499,10 @@ export default {
     },
     addNewTransaction(item) {
       const orgBillingRec = item.orgRec
-      const { charge, rate, description, author } = item
+      const { charge, decimalCharge, rate, description, author } = item
```
Comment (luthian) [inline src/components/billingRecord/IFXBillingRecordList.vue]: Is `decimalCharge` ever likely to be in the same transaction as `charge`?

Comment (aaronk) [reply src/components/billingRecord/IFXBillingRecordList.vue]: No. I'll take that out

**Pull request merged**